Src/AutoFixture/Kernel/OmitRecursionGuard.cs

```diff
@@ -1,34 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Collections;
-
-namespace Ploeh.AutoFixture.Kernel
-{
-    public class OmitRecursionGuard : RecursionGuard
```

Comment:
I've checked. I get some warnings about using obsolete members, which makes sense, given that it's not in the project.